tests/test_os.py

```diff
@@ -95,6 +110,8 @@ class TestOperatingSystemNegative(object):
         ),
         'python -c "import platform;print(\',\'.join('
         'platform.linux_distribution()))"': (1, '', 'SyntaxError'),
+        'uname -r ; uname -v ; uname -m': (1, '', 'SyntaxError'),
```

Comment:
I don't think that `uname` or `date` command can raise any `SyntaxError` , so I believe that these tests are redundant here.
You can either remove them, or think about something more likely happening for these two commands.